SimTracker/SiPhase2Digitizer/python/TBeamTest_cfi.py

```diff
@@ -0,0 +1,35 @@
+import FWCore.ParameterSet.Config as cms
+
+tbeamTest = cms.EDAnalyzer("TBeamTest",
```

Comment:
@suchandradutta - it looks like this analyzer is not in PR nor in the IB. I think, it would be a good idea to add only "working" python configurations.